package.json

```diff
@@ -96,6 +101,7 @@
     "multidep": "^2.0.2",
     "prettier": "^2.2.1",
     "protractor": "^7.0.0",
+    "semantic-release": "^17.4.2",
```

Comment:
Should we also all `commitizen`, `cz-conventional-changelog` and configure it as per [this example](https://github.com/jan-molak/authenticator-browser-extension/blob/master/package.json#L103-L107)?

We _could_ also add `git-cz` to our npm scripts - like [here](https://github.com/jan-molak/authenticator-browser-extension/blob/master/package.json#L24).

Reply:
Fine by me.